docs/standaard/catalogi/release_notes.md

```diff
@@ -6,6 +6,7 @@ layout: page-with-side-nav
 ---
 # Release Notes Catalogi API
 
+## Versies 1.3.1/1.2.1
```

Comment:
Spatie voor en na de slash maakt het naar mijn mening overzichtelijker.

Versies 1.3.1/1.2.1 - > Versies 1.3.1 / 1.2.1

Geldt ook voor andere plekken.

Reply:
Spaties ingevoegd - ook bij release notes voor Documenten API in PR #2423